server/service/src/report/report_service.rs

```diff
@@ -159,80 +159,11 @@ pub trait ReportServiceTrait: Sync + Send {
         let file = file_service
             .store_file(
                 &format!("{}_{}.html", now.format("%Y%m%d_%H%M%S"), report_name),
-                self.format_html_document(document).as_bytes(),
+                document.document.as_bytes(),
```

Comment:
this only prints the body section of the report - the reports I'm using have headers which should be repeated on every page ( which is why this code is here ).

Found that the page formatting is not needed though. In chrome, this works for me:

```

fn format_html_document(&self, document: GeneratedReport) -> String {
        format!(
            "<html>
  <body>
      <table class=\"paging\">
      <thead>
        <tr>
          <td>{}</td>
        </tr>
      </thead>
      <tbody>
        <tr>
          <td>{}</td>
        </tr>
      </tbody>
      <tfoot>
        <tr>
          <td>{}</td>
        </tr>
      </tfoot>
    </table>
  </body>
</html>",
            document.header.unwrap_or("".to_string()),
            document.document,
            document.footer.unwrap_or("".to_string())
        )
    }
```

Reply:
I do the html formatting for the html to html and html to pdf printing now.

Also cleaned up the trait a bit and added some doc comments.

Note, at the moment we always assume html reports. I made this clearer in the service methods. I left the graphql methods as it but I think since they are not used in prod yet we are still able to rename them as well?